contracts/CentralizedAppealableArbitrator.sol

```diff
@@ -178,9 +178,9 @@ contract CentralizedAppealableArbitrator is Arbitrator {
 
     struct DisputeStruct {
         IArbitrable arbitrated;
+        bool isAppealed;
         uint256 choices;
         uint256 fee;
-        bool isAppealed;
         uint256 rulingAppealTimeOut;
         uint256 ruling;
         DisputeStatus status;
```

Comment:
I am not sure this is optimium.

Maybe

```
struct DisputeStruct {
        uint256 choices; // 1 slot	     
        uint256 fee;	 // 1 slot	    	
        uint256 rulingAppealTimeOut; // 1 slot		       
        uint256 ruling; // 1 slot	
        IArbitrable arbitrated;	
        bool isAppealed;
        DisputeStatus status; // 1 slot for the last one
 }
 ```

Reply:
Both will work, but won't yours be

```
struct DisputeStruct {
        uint256 choices; // 1 slot	     
        uint256 fee;	 // 1 slot	    	
        uint256 rulingAppealTimeOut; // 1 slot		       
        uint256 ruling; // 1 slot	
        IArbitrable arbitrated; // 1 slot
        bool isAppealed;
        DisputeStatus status; // 1 slot for the last one
 }
```

Reply:
Both `IArbitrable arbitrated` and `bool isAppealed` will acquire 1 slot together as far as I am concerned. what you think?

Reply:
@n1c01a5 Yes, you are right. Sorry, I really didn't noticed that `DisputeStatus status` which acts as an enum. Let me correct that..Thanks!